### ISSUE TITLE
Obsolete with TYPO3v11

### ISSUE THREAD

**Issue by mbrodala** [sabbelasichon/t3_interpolate_log_processor#6]
AFAIS this extension is obsolete since TYPO3v11, see [Important: #94315 - Use proper PSR-3 logging messages and context](https://docs.typo3.org/c/typo3/cms-core/main/en-us/Changelog/11.3/Important-94315-UseProperPSR-3LoggingMessagesAndContext.html)

**Comment by sabbelasichon** [comment]
Nice. Was not aware of it. I have developed it for v10 and the Messenger Integration. I am going to archive the repo.

**Comment by sabbelasichon** [comment]
As far as i can see the integration within TYPO3 does only support primitive values and misses some other formatting features. And IMHO i don´t like the integration within the AbstractWriter class. I think it would better fit as a single Processor just as in Monolog https://github.com/Seldaek/monolog/blob/main/src/Monolog/Processor/PsrLogMessageProcessor.php

**Comment by mbrodala** [comment]
It makes sense to only support primitive values since you can never know where log messages are sent to (e.g. a message queue)

Aside from that maybe create a ticket on Forge with a description how and why the logging in TYPO3 could be improved here.

**Comment by sabbelasichon** [comment]
@mbrodala That´s true. But i meant a better transformation from a complex type to a primitive type as the MonologProcessor does it.
